mne/viz/evoked.py

```diff
@@ -1024,6 +1024,9 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     if picks is not None:
         pick_names = [evoked.info['ch_names'][pick] for pick in picks]
         evoked.pick_channels(pick_names)
+    else:
+        evoked.pick_types(eeg=True, eog=True, seeg=True, meg=True)
```

Comment:
Good point @kingjr.

Let's do that in a later PR.

(IMO, eog should be included if locs are available, but if the default is to not count eog under data channels, that is the convention we should, even if I disagree with it, follow.)